src/app/src/sensors.json

```diff
@@ -23,7 +45,29 @@
         "Location": "Wissahickon",
         "Name": "USGS 01474000 Wissahickon Creek at Mouth, Philadelphia, PA",
         "Id": "01474000",
-        "ApiAccess": true
+        "ApiAccess": true,
+        "HealthyRanges": {
+            "TURBIDITY": {
+                "lower": 0,
+                "upper": 29
```

Comment:
Not sure where is the appopriate place to put this suggestion but it looks like Lodash has an `inRange` function -- https://lodash.com/docs/4.17.11#inRange -- which takes two args and returns a bool. Since each of these settings has that, I wonder if it'd make sense to create a function like

```
const makeHealthyRangeFunction = (lower, upper) => (value) => inRange(value, lower, upper);
```

and then create small helpers like

```
const inHealthyTurbidityRange = makeHealthyRangeFunction(0, 29);
const inHealthyOxygenRange = makeHealthyRangeFunction(5, 6);
```

etc

Reply:
Thanks. I was actually looking for a function like this earlier, but I missed it in the docs. However, I think the current implementation is simple enough to not warrant bringing in this additional function.